client/src/index.js

```diff
@@ -3,41 +3,44 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter as Router,Route, Link} from 'react-router-dom';
 import './index.css';
 import VideoPage from './components/VideoPage';
+import LoginPage from './components/LoginPage';
 import * as serviceWorker from './serviceWorker';
-
-import "../node_modules/lucid-ui/dist/lucid.css";
-
+import { Tabs } from 'lucid-ui';
+import {withRouter} from 'react-router-dom';
 
 
 
 class App extends React.Component {
-
   render(){
     return(
-    <Router>
-      <div>
-        <Route path="/watch">
-          <VideoPage />
-        </Route>
-        <Route path="/login">
-          <h1> pls login </h1>
-        </Route>
-        <Route exact path="/">
+      <Router>
         <div>
-          <Link to="/watch">WatchVideo</Link>
+
+
+        <div class="tab">
+          <Link className="tablinks" to="/">Home</Link>
+          <Link className="tablinks"  to="/watch">Watch</Link>
+          <Link className="tablinks"  to="/login">Login</Link>
```

Comment:
This seems that it goes against our design. the homepage is the log in page, not a tab to be directed, and what happens when they log on? the tab would have to change to log out.

Reply:
Sam will be re-organizing the page layout in later PR